src/GitVersion.Core/VersionConverters/AssemblyInfo/AssemblyInfoFileUpdater.cs

```diff
@@ -13,7 +13,7 @@ public interface IAssemblyInfoFileUpdater : IVersionConverter<AssemblyInfoContex
     {
     }
 
-    public class AssemblyInfoFileUpdater : IAssemblyInfoFileUpdater
+    public sealed class AssemblyInfoFileUpdater : IAssemblyInfoFileUpdater
```

Comment:
When class wasn't inherited I went with the option of sealing the class but we could also go for the disposable pattern implementation.

Note that this change could be considered as breaking change if you consider this was part of your API.